invenio_rdm_records/config.py

```diff
@@ -94,7 +94,6 @@ def always_valid(identifier):
     "url": {"label": _("URL"), "validator": idutils.is_url, "datacite": "URL"},
     "urn": {"label": _("URN"), "validator": idutils.is_urn, "datacite": "URN"},
     "w3id": {"label": _("W3ID"), "validator": always_valid, "datacite": "w3id"},
```

Comment:
Will it be that now all IDs will be identified as `w3id`, since it's the last entry with `always_valid`?

I'm not sure what the logic for scheme detecting is for related identifiers...

Reply:
I can't validate this right now. 
I had a quick look and it seems that this "fallback" does not happen, but I want to be more thorough with the analysis. I will do it in a separate [task](https://github.com/inveniosoftware/invenio-rdm-records/issues/1584)